circuits/cpp/src/aztec3/circuits/abis/combined_accumulated_data.hpp

```diff
@@ -235,6 +277,14 @@ template <typename NCT> std::ostream& operator<<(std::ostream& os, CombinedAccum
               << accum_data.public_call_stack << "\n"
               << "new_l2_to_l1_msgs:\n"
               << accum_data.new_l2_to_l1_msgs << "\n"
+              << "encrypted_logs_hash:\n"
```

Comment:
hmm I thought adam's msgpack work didn't require this anymore as seen in `PublicCircuitPublicInputs` class

Reply:
It probably should be removed but maybe @ludamad plans on doing so in his other [msgpack PR](https://github.com/AztecProtocol/aztec-packages/pull/634) he is working on.